src/main/java/nl/uu/cs/ape/domain/BioToolsAPI.java

```diff
@@ -71,30 +86,34 @@ public static JSONArray readListOfTools(String filePath) throws IOException {
 	 * @return JSONArray with the tool annotations.
 	 * @throws IOException - If the file cannot be read or written.
 	 */
-	public static JSONArray getToolsFromDomain(String domainName) throws IOException {
+	private static JSONObject getToolsFromDomain(String domainName) throws IOException {
 		JSONArray toolAnnotations = null;
 		if (!domainName.equals("")) {
```

Comment:
Is there a reason for not using isEmpty here?

Reply:
No specific reason, I updated the methods in the upcoming commit